File: c#/1020.cs
using System;

class URI {

    static void Main(string[] args) {

        string input_time = Console.ReadLine();

        int time = int.Parse(input_time);

        int year = time / 365;

        int mounth = (time - year * 365) / 30;

        int day = time - year * 365 - mounth * 30;

        Console.WriteLine($"{year} ano(s)\n{mounth} mes(es)\n{day} dia(s)");
        
    }    
}

File: c++/1020.cpp
#include <stdio.h>
 
int main() {

    int time, year, mounth, day;

    scanf("%d", &time);

    year = time / 365;

    mounth = (time - year * 365) / 30;

    day = time - year * 365 - mounth * 30;

    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", year, mounth, day);
 
    return 0;
}